ruby_on_rails/rails_sprinkles/turbo.md

```diff
@@ -319,7 +319,7 @@ The final piece of Turbo is something that you don't need to know much about for
 
 <div class="lesson-content__panel" markdown="1">
 
-#### Single Page Applications
+#### Single page applications
```

Comment:
```suggestion
#### Single Page Applications
```

Reply:
feel free to `commit suggestion` on this one as well if you want. You can do that with each suggestion a maintainer makes to avoid having to copy it to your local doc but instead commit it directly.